### DISCUSSION TITLE
Sync config center error

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11853] [Q&A]
skywalking version 9.7.0

A screenshot error occurs when configuring the consul Configuration Center.

<img width="1854" alt="企业微信截图_d8cff2dd-36d0-4e82-8ddd-101d0de1c145" src="https://github.com/apache/skywalking/assets/31468018/bc07a2ef-0d07-4074-a3a9-aee657bc8c77">

**Comment by 1450793561** [comment]
Is the configuration successful or unsuccessful?

**Reply by weixiang1862** [reply]
The value of `core.default.apdexThreshold` is a map structure, not a single number value, it is same as [service-apdex-threshold.yml](https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/service-apdex-threshold.yml).

**Comment by ranxuxin001** [comment]
how to verify the apdex configuration is configured correct?
